blazor/dropdown-list/how-to/disable-the-dropdownlist-popup-items-based-on-selection.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: Disable the DropDown List Popup items based on selection | Syncfusion
+description: Learn how to disable the DropDown List popup items based on the selection in the Syncfusion Blazor DropDown List component.
+platform: Blazor
+control: DropDown List
+documentation: ug
+---
+
+# How to disable the DropDown List popup items based on selection
+
+The following example demonstrates how to disable the DropDown List popup items based on the selection in the DropDown List component. This can be achieved by adding the e-disabled class to the selected list.
+
+```cshtml
+@using Syncfusion.Blazor
```

Comment:
Need to move the code to a separate file. Follow all the guidelines.

https://syncfusion.sharepoint.com/sites/Install/SitePages/Support%20for%20loading%20code%20snippet%20from%20a%20file.aspx

https://github.com/syncfusion-content/blazor-docs/tree/hotfix/hotfix-v19.3.0.43#general-guidelines